api/src/modules/projects/projects.controller.ts

```diff
@@ -23,7 +23,19 @@ export class ProjectsController {
   @TsRestHandler(projectsContract.getProjects)
   async getProjects(): ControllerResponse {
     return tsRestHandler(projectsContract.getProjects, async ({ query }) => {
-      const data = await this.projectsService.findAllPaginated(query);
+      // The following filters do not work out of the box with the BaseService implementation
```

Comment:
I see what u mean about being hackish xD.  You probably need more context as to why I approached other filters this way, but overall, I think we can just pass the query object here. I'll try to add more context below